scripts/README.rst

```diff
@@ -0,0 +1,145 @@
+Misc. scripts
+=============
+
+These scripts are used for many different purposes throughout RPCO.
+Below is a description of all of the scripts in this directory.
+
+alarmparser.py
+--------------
+
+alarmparser.py is used by the rpc-maas-tool.py script to check the MaaS
+alarm syntax on each check file in a system. This file was generated by
+Waxeye Parser Generator and should not be modified.
+
+checksha.py
+-----------
+
+This helper script checks to see if the supplied commit sha is contained
+in a specific wheel of a project i.e. neutron
+
+deploy-rpc-playbook.sh
+----------------------
+
+This script is invoked by deploy.sh, and runs the RPCO specific
+playbooks using the run\_ansible function implemented in functions.sh
+
+deploy.sh
+---------
+
+NOTE: This script is currently being reworked as part of
+https://github.com/rcbops/u-suk-dev/issues/834. TODO: (alextricity25)
+Update this document once the above card is complete
+
+By far the most used and important script in this directory. The
+deploy.sh script is used to deploy RPCO with certain features turned on
+or off. These are toggled via environment variables described
+`here <https://github.com/rcbops/rpc-openstack/tree/master#environment-variables-for-deploysh>`__.
+
+When adding a new feature into RPCO, there might be a need to add a
+corresponding DEPLOY variable into this script so the feature can be
+toggled on or off. Please follow the same pattern outlined in this
+script, and enclose any instructions specific to deploying the new
+feature with a conditional similar to what is done
+`here <https://github.com/rcbops/rpc-openstack/blob/master/scripts/deploy.sh#L226>`__
+for ceph.
+
+f5-config.py
+------------
+
+NOTE: The f5-config.py script is currently being reworked as part of
```

Comment:
I'll make this a part of https://github.com/rcbops/u-suk-dev/issues/836 and add a note there